source/install/transport-encryption/config-cluster.rst

```diff
@@ -0,0 +1,231 @@
+=============================================
+Configuring Cluster Transport Encryption (E20)
+=============================================
+
+Mattermost is able to encrypt the messages sent within the cluster of a deployment
+using SSH tunneling. The guide walks through the deployment of this solution on
+Ubuntu 18.04, but it can be adapted for any Linux operating system.
+
+While this document only describes the configuration of a three-node cluster, it is
+by no means limited to that amount.
+
+## Prerequisites
```

Comment:
The header formatting should be corrected, and the list formatting is not rendering correctly.